test/MOIWrapper.jl

```diff
@@ -14,17 +14,19 @@ MOIU.@model(SCSModelData,
             (MOI.ScalarAffineFunction,),
             (MOI.VectorOfVariables,),
             (MOI.VectorAffineFunction,))
-const optimizer = MOIU.CachingOptimizer(SCSModelData{Float64}(), SCS.Optimizer())
+for T in [SCS.Direct, SCS.Indirect]
+    optimizer = MOIU.CachingOptimizer(SCSModelData{Float64}(), SCS.Optimizer(linear_solver=T))
 
-# linear9test needs 1e-3 with SCS < 2.0 and 5e-1 with SCS 2.0
-# linear2test needs 1e-4
-const config = MOIT.TestConfig(atol=1e-4, rtol=1e-4)
+    # linear9test needs 1e-3 with SCS < 2.0 and 5e-1 with SCS 2.0
```

Comment:
Why do we care about tolerances for an SCS version that we don't use anymore?

Reply:
The tolerance needed for SCS < 2.0 is mentionned to show that this is a regression

Reply:
It's unlikely that this code will handle SCS < 2.0 in the future, so this isn't the right place to discuss regressions.